tests/reference/asr-expr7-480ba2f.stderr

```diff
@@ -0,0 +1,11 @@
+warning: Could have used '**' instead of 'pow'
```

Comment:
I think this should say `style`, not `warning`. You might need to recompile and update tests.

Reply:
Done.  :)